src/lib/Behat/Component/UniversalDiscoveryWidget.php

```diff
@@ -166,13 +166,7 @@ public function selectBookmark(string $bookmarkName): void
     public function editSelectedContent(): void
     {
         $this->getHTMLPage()->setTimeout(self::SHORT_TIMEOUT)->find($this->getLocator('editButton'))->click();
-        $iframeLocator = $this->getLocator('iframe');
-        $script = sprintf("document.querySelector('%s').setAttribute('name','editIframe')", $iframeLocator->getSelector());
-        $this->getHTMLPage()->setTimeout(self::SHORT_TIMEOUT)->waitUntilCondition(
-            new ElementExistsCondition($this->getHTMLPage(), $iframeLocator)
-        );
-        $this->getHTMLPage()->executeJavaScript($script);
-        $this->getSession()->switchToIFrame('editIframe');
+        $this->switchIntoUDWIframe('editIframe');
```

Comment:
```suggestion
        $this->switchIntoContentEditIframe();
```
I think we can write this function without the need of knowing if the iframe will be an edit or create iframe.
Also I think that the function name should be related to `Content` somehow - because for me it's the Content create/edit view that's embedded inside the iframe. Maybe `switchIntoContentOnTheFlyIframe`?